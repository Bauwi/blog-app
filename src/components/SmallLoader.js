import React from 'react';

export default () => (
  <div className="loader loader--small">
    <img className="loader__image" src="/images/loader.gif" alt="Loading..." />
  </div>
);
