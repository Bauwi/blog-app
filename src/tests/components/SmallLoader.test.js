import React from 'react';
import { shallow } from 'enzyme';
import SmallLoader from '../../components/SmallLoader';

test('should render SmallLoader component properly', () => {
  const wrapper = shallow(<SmallLoader />);
  expect(wrapper).toMatchSnapshot();
});
