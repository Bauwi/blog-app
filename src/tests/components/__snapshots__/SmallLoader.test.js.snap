// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SmallLoader component properly 1`] = `
<div
  className="loader loader--small"
>
  <img
    alt="Loading..."
    className="loader__image"
    src="/images/loader.gif"
  />
</div>
`;
